src/katello/repos.py

```diff
@@ -36,7 +37,10 @@ def report_enabled_repos(consumer_id, report):
         error_message(str(error))
 
 
-def upload_enabled_repos_report(report):
+def upload_enabled_repos_report(report, force=False):
```

Comment:
At a glance it appears this change is affecting `test_katello.test_plugin.TestCertificateChanged` which seems to hang up. I think it's this method since it's used by the plugin